### ISSUE TITLE
Support changing resolution/aspect ratio pojavlauncher 

### ISSUE THREAD

**Issue by S0rk-jpg** [PojavLauncherTeam/PojavLauncher#4317]
### Is this request related to a bug?

As I already made a similar request in 2021 I will try to simplify. I wanted them to make a resolution change option not like the one that already exists in pojav but one that changes the aspect ratio between 4:3 and 16:8 for example 

![Screenshot_20230701-161335](https://github.com/PojavLauncherTeam/PojavLauncher/assets/82401412/405128eb-b4b5-4c89-a768-36d6be0678dd)

### Describe the request.

An option outside in pojavlauncher that changes the aspect ratio 
16:9
5:3
4:3
5:4
4:5

Option can be scroll like search versions or left in order
![Screenshot_20230701-162357-004](https://github.com/PojavLauncherTeam/PojavLauncher/assets/82401412/96065ff9-67e9-4d4c-87fb-f6d815ddec6b)

### Anything else?

As far as I remember in 2021 they had no interest in this subject I came back again to find out if in 2023 you could pay attention to this (don't judge me for lack of knowledge if this is not possible)

**Comment by GaucovyAero** [comment]
idk ima make it worth considering, but this is very much a low priority thing

**Comment by Mathias-Boulay** [comment]
why would anyone need to reduce their screen size ?

**Comment by GaucovyAero** [comment]
to record videos for youtube in 16:9 i guess

**Comment by PSgamer20x** [comment]
> why would anyone need to reduce their screen size ?

Yes we really need 16:9 screen it's good for making videos fr add it

**Comment by Mathias-Boulay** [comment]
We're open for PR on this issue

**Comment by K4sum1** [comment]
It would be cool to be able to crop the game to 4:3 on a 16:9/18:9 phone, and 16:9 on a taller phone, then have the controls purely in the non-game portion. Be able to play the game without the controls covering any of it up, with the added benefit of less game being rendered equaling more performance and/or battery life.
